css-backgrounds-3/Overview.bs

```diff
@@ -2504,7 +2505,8 @@ Layering, Layout, and Other Details</h4>
   and the <i>inner shadows</i> of an element are drawn immediately above the background of that element
   (below the borders and border image, if any).
 
-  <p>If an element has multiple boxes, all of them get drop shadows,
+  <p>Drop shadows are only applied to the [=principal box=].
+  If that box has multiple [=fragments=], then all of them get shadows,
```

Comment:
This is incorrect. How box-shadow is handled on fragments depends on `box-decoration-break`, see https://www.w3.org/TR/css-break-4/#break-decoration

Reply:
Isn't this addressed in the next sentence?

>  but shadows are only drawn where borders would also be drawn;
>  see 'box-decoration-break'.

Reply:
No, it's not that we have multiple shadows some parts of which aren't drawn. box-decoration-break determines whether the entire box gets shadowed and then sliced or the fragments get shadowed. If you offset the shadow by more than the width of the first fragment and then blur it, for example, the first fragment won't have a shadow attached, but there will be a shadow as wide as the combined fragments, blurred as a single unit, attached to the second fragment.